<launch>
    <group ns="servogroup12">
    <node pkg="ftservoControl" type="singleServoControl" name="singleServoControl" output="screen" >
        <param name="id_up" type="int" value="2" />
        <param name="id_down" type="int" value="1" />
        <param name="source_frame" type="string" value="EstimationfromcamA" />
        <param name="target_frame" type="string" value="camA" />
        <param name="cam" type="string" value="camA" />
        <param name="serial" type="string" value="/dev/ttyUSB0" />
    </node>
    </group>
</launch>
